docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
+.. Enter "Median of bytes" as the panel title, then click *Save*.
 
-.. In the *Series color* field, enter *#CA8EAE*, then click *Close*.
+. Click *Save* on the entire dashboard.
 
 [discrete]
-[[configure-the-multiaxis-chart]]
-==== Configure the y-axes
-
-There is a significant difference between the *timestamp per day* and *Unique visitors* data, which makes the *Unique visitors* data difficult to read. To improve the readability, 
-display the *Unique visitors* data along a second y-axis, then change the formatting. When functions contain multiple formats, separate axes are created by default. 
-
-. In the editor, click *Unique visitors* in the line chart layer.
+[[view-the-distribution-of-visitors-by-operating-system]]
+=== View the most frequent values
 
-. For *Axis side*, click *Right*, then click *Close*.
+When you drag a text or IP address field into the *Lens* workspace you will see
+the most frequent values of that field using the *top values* function. The values
+are ranked based on another metric like *Count of records*. Only exact matches count:
+`win 10` and `win xp` are distinct.
 
-[float]
-[[change-the-visualization-type]]
-==== Change the visualization type
+For this tutorial, you have decided to see the most frequently viewed URLs on your website
+ranked by the `Unique count of clientip` metric, but you have not yet determined which chart type to use.
 
-. In the editor, click *Average of bytes* in the area chart layer. 
+. Starting with the default bar chart, drag the `clientip` field onto the *Vertical axis*.
+This will produce the `Unique count of clientip` metric. Do not drop the field into the main workspace,
+because the *top values* function will be used instead.
 
-. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
+. Drag and drop `request.keyword` to the main workspace, and *Lens* will add `Top value of request.keyword`
+to the *Horizontal axis*.
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_4.png[Multiaxis chart]
+image::images/lens_end_to_end_2_1_1.png[Vertical bar chart with top values of request.keyword by most unique visitors]
 
-[discrete]
-[[lens-legend-position]]
-==== Change the legend position
+This chart is hard to read because the `request.keyword` field contains long text. You could try
+using one of the *Suggestions*, but the suggestions also have issues with long text. Instead, switch
+to a *Table* visualization:
 
-The visualization is done, but the legend uses a lot of space. Change the legend position to the top of the chart.
+. Click *Bar vertical stacked* to open the chart switcher
 
-. From the *Legend* dropdown, select the top position.
+. Click *Table*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_5.png[legend position]
+image::images/lens_end_to_end_2_1_2.png[Table with top values of request.keyword by most unique visitors]
 
-. Click *Save and return*.
+Customize the table even more:
```

Comment:
At this point, users have not customized the table. 

Also, a colon is used to separate two independent clauses when the second explains the first. A period is more appropriate for these introduction-like sentences.